Mer/new checkout 1272 (#1288)
magento/merchdocs: src/quick-tour/path-to-purchase.md

Shopping Cart
:  The [shopping cart]({% link quick-tour/shopping-cart.md %}) lists each item by price and quantity selected, and calculates the subtotal. Shoppers can apply discount coupons, and generate an estimate of shipping and tax charges.

Checkout Page

> :  The [checkout page]({% link quick-tour/checkout-page.md %}) lists different shipping and payment methods, customer can choose as per their choice.

Reviewer: ```suggestion
:  The [checkout page]({% link quick-tour/checkout-page.md %}) allows the customer to choose from different shipping and payment methods and complete their order.
```
